tests/internal/cluster/cluster.go

```diff
@@ -171,3 +179,130 @@ func ExecCmdWithStdout(
 
 	return outputMap, nil
 }
+
+// ExecCmdWithRetries executes a command on the provided client on each node matching nodeSelector,
+// retrying on internal errors
+// "retries" times with a "interval" duration between retries, and ignores the stdout.
+func ExecCmdWithRetries(client *clients.Settings, retries uint,
+	interval time.Duration, nodeSelector, command string) error {
+	glog.V(90).Infof("Executing command '%s' with %d retries and interval %v. Node Selector: %v",
+		command, retries, interval, nodeSelector)
+
+	for retry := range retries {
```

Comment:
It looks like wait PollImmediate* pattern. Please use wait functions instead of sleep

Reply:
Fixed